packages/adapter-pocketbase/tests/index.test.ts

```diff
@@ -0,0 +1,123 @@
+import { runBasicTests } from "@next-auth/adapter-test"
+import { PocketBaseAdapter } from "../src"
+import type {
+  AdapterAccount,
+  VerificationToken,
+  AdapterUser,
+  AdapterSession,
+} from "next-auth/src/adapters"
+import {
+  PocketBaseAccount,
+  PocketBaseSession,
+  PocketBaseUser,
+  PocketBaseVerificationToken,
+} from "../src/pocketbase.types"
+
+import "cross-fetch/polyfill"
+
+import Pocketbase from "pocketbase"
+const pb = new Pocketbase("http://127.0.0.1:8090")
+
+runBasicTests({
+  adapter: PocketBaseAdapter(pb),
+  db: {
+    async session(sessionToken) {
+      let pb_session: any
+      try {
+        pb_session = await pb
+          .collection("next_auth_session")
+          .getFirstListItem<PocketBaseSession>(`sessionToken="${sessionToken}"`)
+      } catch (_) {
+        return null
+      }
+
+      if (pb_session.sessionToken !== sessionToken)
```

Comment:
This looks impossible considering a row is taken from a DB by 'sessionToken'

Reply:
Hey! Thanks for giving the code a peruse. 

Yeah definitely just a paranoia check. I was getting weird jest assertions when rerunning the tests due to getAccountByEmail, ACTUALLY having multiple matching emails in the table. So I did this to make sure it could NOT be done by any other get test. Definitely not needed though, and I'll remove later.